config/oot-j/config.yml

```diff
@@ -0,0 +1,154 @@
+# See config.example.yml for documentation.
+name: oot-vc
```

Comment:
I wonder if we should call this something other than "oot"-vc if we plan to support other games. Maybe just "vc"? It doesn't seem to hard to change later and right now it is only oot, so maybe not worth worrying about not

Reply:
I think on oot-gc it's called "oot-gc", iirc this is the name the build will have 🤔

Reply:
yeah I think this is fine for now